{"name": "test_go_to_order_history", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <tests.test_profile.TestProfile object at 0x000002BCC05FAD50>\ndriver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"cefcfadf-95e8-49b0-8f15-f42829035aa9\")>\ncreate_and_login_user = {'accessToken': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY4MzFkNDliOWVkMjgwMDAxYjVmOTdmNCIsImlhdCI6MTc0O...i27LC4bzS-ye3O9Q-bjQnVfQkKVzARlkAzgycs', 'email': 'test_rnipw@yandex.ru', 'name': 'mljUNljZ', 'password': 'jkLEb8GMnW'}\n\n    @allure.story(\"Переход в раздел История заказов\")\n    def test_go_to_order_history(self, driver, create_and_login_user):\n        driver.get(login_url)\n        login = LoginPage(driver)\n        login.login_user(create_and_login_user[\"email\"], create_and_login_user[\"password\"])\n        profile = ProfilePage(driver)\n        profile.go_to_profile()\n>       profile.go_to_order_history()\n\ntests\\test_profile.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\profile_page.py:34: in go_to_order_history\n    self.click(ProfilePageLocators.ORDER_HISTORY_BUTTON)\npages\\base_page.py:54: in click\n    element = WebDriverWait(self.driver, 10).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"cefcfadf-95e8-49b0-8f15-f42829035aa9\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x000002BCC0657560>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Wait until the method returns a value that is not False.\n    \n        Calls the method provided with the driver as an argument until the\n        return value does not evaluate to ``False``.\n    \n        Parameters:\n        -----------\n        method: callable(WebDriver)\n            - A callable object that takes a WebDriver instance as an argument.\n    \n        message: str\n            - Optional message for :exc:`TimeoutException`\n    \n        Return:\n        -------\n        object: T\n            - The result of the last call to `method`\n    \n        Raises:\n        -------\n        TimeoutException\n            - If 'method' does not return a truthy value within the WebDriverWait\n            object's timeout\n    \n        Example:\n        --------\n        >>> from selenium.webdriver.common.by import By\n        >>> from selenium.webdriver.support.ui import WebDriverWait\n        >>> from selenium.webdriver.support import expected_conditions as EC\n    \n        # Wait until an element is visible on the page\n        >>> wait = WebDriverWait(driver, 10)\n        >>> element = wait.until(EC.visibility_of_element_located((By.ID, \"exampleId\")))\n        >>> print(element.text)\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\nvenv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:146: TimeoutException"}, "steps": [{"name": "Ввод email: 'test_rnipw@yandex.ru'", "status": "passed", "parameters": [{"name": "email", "value": "'test_rnipw@yandex.ru'"}], "start": 1748096156372, "stop": 1748096156425}, {"name": "Ввод пароля", "status": "passed", "parameters": [{"name": "password", "value": "'jkLEb8GMnW'"}], "start": 1748096156425, "stop": 1748096156443}, {"name": "Нажатие на кнопку 'Войти'", "status": "passed", "start": 1748096156443, "stop": 1748096156726}, {"name": "Клик по кнопке 'Личный кабинет'", "status": "passed", "start": 1748096156726, "stop": 1748096157006}, {"name": "Клик по кнопке 'История заказов'", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"C:\\Users\\slipk\\PycharmProjects\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\slipk\\PycharmProjects\\Diplom_3\\pages\\profile_page.py\", line 34, in go_to_order_history\n    self.click(ProfilePageLocators.ORDER_HISTORY_BUTTON)\n  File \"C:\\Users\\slipk\\PycharmProjects\\Diplom_3\\pages\\base_page.py\", line 54, in click\n    element = WebDriverWait(self.driver, 10).until(\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\slipk\\PycharmProjects\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 146, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1748096157006, "stop": 1748096167058}], "start": 1748096155764, "stop": 1748096167059, "uuid": "ea6a1b27-bcdf-464d-8e08-7391daee357d", "historyId": "64dd331b350b27103be248cd6e4fd5c5", "testCaseId": "64dd331b350b27103be248cd6e4fd5c5", "fullName": "tests.test_profile.TestProfile#test_go_to_order_history", "labels": [{"name": "feature", "value": "Личный кабинет"}, {"name": "story", "value": "Переход в раздел История заказов"}, {"name": "tag", "value": "@pytest.mark.usefixtures('driver')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_profile"}, {"name": "subSuite", "value": "TestProfile"}, {"name": "host", "value": "DESKTOP-05J7DPF"}, {"name": "thread", "value": "23876-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_profile"}]}